tardis/base.py

```diff
@@ -16,27 +16,39 @@ def run_tardis(
     specific=None,
 ):
     """
-    This function is one of the core functions to run TARDIS from a given
-    config object.
+    Run TARDIS from a given config object.
 
-    It will return a model object containing
+    It will return a model object containing the TARDIS Simulation.
 
     Parameters
     ----------
     config : str or dict or tardis.io.config_reader.Configuration
         filename of configuration yaml file or dictionary or TARDIS Configuration object
-    atom_data : str or tardis.atomic.AtomData
-        if atom_data is a string it is interpreted as a path to a file storing
+    atom_data : str or tardis.atomic.AtomData, optional
+        If atom_data is a string it is interpreted as a path to a file storing
         the atomic data. Atomic data to use for this TARDIS simulation. If set to None, the
         atomic data will be loaded according to keywords set in the configuration
-        [default=None]
-    virtual_packet_logging : bool
-        option to enable virtual packet logging
-        [default=False]
+    packet_source : class, optional
+        A custom packet source class or a child class of `tardis.montecarlo.packet_source`
+        used to override the TARDIS `BasePacketSource` class.
+    simulation_callbacks : list of lists, optional
+        Set of callbacks to call at the end of every iteration of the Simulation.
+        The format of the lists should look like:
+        [[callback1, callback_arg1], [callback2, callback_arg2], ...]
+        Where the callback function signature should look like:
+        callback_function(simulation, extra_arg1, ...)
+    virtual_packet_logging : bool, default False, optional
+        Option to enable virtual packet logging.
+    log_state : str, optional
+        Set the level of the TARDIS logger. Options include `NOTSET`, `DEBUG`, `INFO`, `WARNING`, `ERROR`, and `CRITICAL`.
+        Default level of the logger is set to CRITICAL.
+    specific : bool, optional
+        If True, only show the log messages from a particular log level, set by `log_state`,
+        If False, the logger shows log messages belonging to the level set and all levels above it in severity.
 
     Returns
     -------
-    Simulation
+    simulation : tardis.simulation.Simulation
```

Comment:
If there is only one variable being returned, don't specify the variable name but only type (see pandas docstyle guide I shared)